srcpkgs/pinta/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'pinta'
+pkgname=pinta
+version=1.7
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake autoconf pkg-config"
+makedepends="mono-devel gtk-sharp2 intltool mono-addins"
```

Comment:
also `depends="mono-addins"`

Reply:
Okay, moved `mono-addins` to `depends`

Reply:
Moved back to `makedepends` as that makes the template pass `./xbps-src pkg` build.